File: JavaExtras/GRPC/server/src/main/java/app/entity/Emp.java
package app.entity;

import java.util.Date;

import jakarta.persistence.Entity;
// import jakarta.persistence.GeneratedValue;
// import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;


@Entity
@Table(name="EMP")
public class Emp{
    @Id
    // @GeneratedValue(strategy=GenerationType.IDENTITY)
    public double empno;
    public String ename;
    public double sal;
    public Date hiredate;
    public int mgr;
    public String job;
    public double comm;
    public double deptno;
}

File: DotNet_master/DotNetAssignments/c#AssignAkash/encore_ahh/efcore/data/emp.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace efcore.data;

[Table("EMPLOYEES45")]
public class Employee
{
    [Column("empno")]
    public decimal Id { get; set; }
    [Column("ename")]
    public string Name { get; set; }
    [Column("Job")]
    public string Job { get; set; }
    [Column("mgr")]
    public decimal Mgr { get; set; }
    [Column("Hire Date")]
    public DateTime Hiredate { get; set; }
    [Column("Salary")]
    public decimal Salary { get; set; }
    [Column("Comm")]
    public decimal Comm { get; set; }

    [ForeignKey("Department45")]
    [Column("deptno")]
    public decimal DepartmentId { get; set; }
}